src/actions/MapActions.js

```diff
@@ -117,6 +118,12 @@ export const openMap = (mapId) => {
         });
       }, 1000);
 
+      if (sessionStorage.getItem("currentMapId")) {
+        sessionStorage.removeItem("currentMapId");
+      }
+
+      sessionStorage.setItem("currentMapId", mapId);
```

Comment:
as well as setting the item here, I think we should also remove it in the `newMap` action. Otherwise, if we open a new untitled map and the refresh the page will return to the saved map.